src/uct/ib/base/ib_md.c

```diff
@@ -1482,6 +1482,23 @@ ucs_status_t uct_ib_md_open_common(uct_ib_md_t *md,
     md->super.ops       = &uct_ib_md_ops;
     md->super.component = &uct_ib_component;
 
+    if ((md->config.mr_relaxed_order == UCS_CONFIG_ON) && !IBV_ACCESS_RELAXED_ORDERING) {
+        ucs_warn("relaxed order memory access requested but not supported");
+    } else if (md->config.mr_relaxed_order == UCS_CONFIG_AUTO) {
+#ifdef __aarch64__
+        ucs_aarch64_cpuid_t cpuid;
+        ucs_aarch64_cpuid(&cpuid);
+
+        if ((cpuid.implementer == 0x46) && (cpuid.architecture == 8)) {
```

Comment:
Why you wound not add some sort of generic query about PCIE ordering preferences in ucs/arch ?

Reply:
in UCS?